mysql/mysql.ml

```diff
@@ -391,7 +391,7 @@ module Expr = struct
   let date ?(year = 0) ?(month = 0) ?(day = 0) = fun _ ->
     Lit (Lit.Date { base_time with year; month; day })
   let datetime ?(year = 0) ?(month = 0) ?(day = 0)
-               ?(hour = 0) ?(minute = 0) ?(second = 0) = fun _ ->
```

Comment:
Shouldn't you actually use that value in `{ base_time with ...` instead of removing it?

Reply:
I removed it because it was unused. Good point tho.